src/main.rs

```diff
@@ -30,7 +30,7 @@ struct Args {
 }
 
 #[derive(StructOpt, Debug)]
-#[structopt(bin_name = "cargo")]
+#[structopt(name = "cargo", raw(setting = "AppSettings::ColoredHelp"))]
```

Comment:
Prior to this change (there is no color):
```
$ ./target/debug/cargo-ebuild ebuild --help
cargo-ebuild 0.1.6-pre
Doug Goldstein <cardoe@cardoe.com>
Generates an ebuild for a given Cargo project

USAGE:
    cargo ebuild [OPTIONS]

OPTIONS:
    -q, --quiet      Silence all output
    -v, --verbose    Verbose mode (-v, -vv, -vvv, etc.)
    -h, --help       Prints help information
    -V, --version    Prints version information
```

With this change (there is still no color):
```
$ ./target/debug/cargo-ebuild ebuild --help
cargo-ebuild-ebuild 0.1.6-pre
Doug Goldstein <cardoe@cardoe.com>
Generates an ebuild for a given Cargo project

USAGE:
    cargo-ebuild ebuild [OPTIONS]

OPTIONS:
    -q, --quiet      Silence all output
    -v, --verbose    Verbose mode (-v, -vv, -vvv, etc.)
    -h, --help       Prints help information
    -V, --version    Prints version information
```

So not only does this not add color but it breaks the help output. I would drop this patch.